.github/workflows/pypi-ci.yml

```diff
@@ -9,69 +12,116 @@ on:
     types:
       - published
 jobs:
-  build_wheels:
-    name: Build wheels [${{ matrix.os }}]
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version:
-          - 3.8
-        os:
-          - ubuntu-20.04
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
-      - run: git fetch --prune --unshallow
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
```

Comment:
3.8 is EOL (see https://endoflife.date/python), unless there is any specific reason to be compatible with such an old version, I would bump the minimum version in CI and pyproject to 3.9 .

Reply:
Totally agree! Tbh I would go directly to 3.10 (for example to update some annotations). 
Makes sense?

Reply:
Ok for me! IsaacLab is past 3.9 now, right? The old Python was a thing of IsaacGym?

Reply:
Exactly!

Reply:
Isaac lab now should be on python 3.10

Reply:
Updated with with [85770e3](https://github.com/ami-iit/liecasadi/pull/16/commits/85770e369d63ddf72f40ffc956a28c0908f1fbda)!